fragdenstaat_de/fds_cms/models.py

```diff
@@ -109,7 +109,14 @@ class DocumentCollectionEmbedCMSPlugin(CMSPlugin):
     collection = models.ForeignKey(
         DocumentCollection, related_name="+", on_delete=models.CASCADE
     )
-    settings = models.TextField(default="{}")
```

Comment:
Did you check if there are any embeds with settings set on the page?

Reply:
Unless @stefanw knows more.

Reply:
It's merged now and about to be deployed, but [there are some mentions of `maxHeight`](https://github.com/search?q=repo%3Aokfde%2Fdjango-filingcabinet%20maxHeight&type=code). It's supposed to switch embeds between 'page mode' and 'embed mode'. Let's see what happens.